staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -67,12 +71,15 @@ func (m *CRConverterFactory) NewConverter(crd *apiextensionsv1.CustomResourceDef
 		converter = &nopConverter{}
 	case apiextensionsv1.WebhookConverter:
 		converter, err = m.webhookConverterFactory.NewWebhookConverter(crd)
-		if err != nil {
-			return nil, nil, err
-		}
-		converter, err = converterMetricFactorySingleton.addMetrics(crd.Name, converter)
-		if err != nil {
+		if errors.As(err, caParseErr) {
```

Comment:
Bundles parsing CABundles is the only non-transient error I could identify that can originate from the a CRD's WebhookClientConfig that I was able to identify. So I've scoped this fix to those error only.

Reply:
this seems really complicated to do at runtime... why would we not validate the CA bundle at CRD write time?